cypress/integration/code-schools.spec.js

```diff
@@ -0,0 +1,16 @@
+describe('code schools', function() {
+  beforeEach(() => {
+    cy.server();
+    cy.route('GET', '/api/v1/code_schools').as('codeSchools');
+
+    cy.visitAndWaitFor('/code_schools');
+
+    cy.get('h1').should('have.text', 'Code Schools');
+  });
+
+  describe('render code school cards', () => {
+    it('should render code school cards', () => {
+      cy.get('div.code_schools_schoolCardsWrapper__SXWXI');
```

Comment:
So this isn't an assertion, and since it's the only line within the scope of the `it`, it's also not a test.

Check out the list of assertions here https://docs.cypress.io/guides/references/assertions.html

You also don't need to target the element so specifically. Although, I'll make a PR now that will allow us to easily follow [cypress best practices in regards to selecting elements](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements).

Something like:
```javascript
describe('code schools', () => {
  beforeEach(() => {
    cy.server();
    cy.route('GET', '/api/v1/code_schools').as('codeSchools');

    cy.visitAndWaitFor('/code_schools');

    cy.get('h1').should('have.text', 'Code Schools');
  });

  it('should render many code school cards', () => {
    // we don't want to make any assertions until AFTER we hit the API and receive a response
    cy.wait('@codeSchools');

    // 50 is arbitrary, but it definitely proves - at a basic level - that the API is communicating
    // and we're rendering content
    cy.get('article').should('have.length.greaterThan', 50);
  });
});
```